javascript/javascript1/week3/seriesLife.js

```diff
@@ -0,0 +1,53 @@
+const seriesDurations = [
+  {
+    title: "Game of Thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+function logOutSeriesText() {
+  const totalLifeMinutes = 80 * 365 * 24 * 60;
+  console.log("Total Life Minutes: " + totalLifeMinutes); 
+  let totalPercentage = 0; 
+
+  for (const series of seriesDurations) {

```

Comment:
Very nice use of `for...of`! You have used `for (let i = 0;...)` in other places, what made you choose differently here? 😄

Generally, I would suggest you stick to this form as often as possible. Not only is it shorter to type out, it is also less error prone! 🙂